src/components/SmallInput/index.tsx

```diff
@@ -0,0 +1,6 @@
+import { ISmallInput } from './interfaces';
+import * as S from './styles';
+
+const SmallInput = (props: ISmallInput) => <S.Input {...props} />;
```

Comment:
I think it'd be a good idea to forward ref in this one

Reply:
I created a new commit. Is this ok now?